project/MultiNode.scala

```diff
@@ -33,7 +33,7 @@ object MultiNode extends AutoPlugin {
   val multiTest = CliOptions.multiNode.ifTrue(MultiJvm / multiNodeTest).getOrElse(MultiJvm / test)
 
   override def trigger = noTrigger
-  override def requires = plugins.JvmPlugin
+  override def requires = plugins.JvmPlugin && MultiJvmPlugin
```

Comment:
This is now required because beforehand the  `.configs(MultiJvm)`  was inadvertently causing the `MultiJvmPlugin` to load before the `MultiNode` plugin.

Now that `.configs(MultiJvm)` is removed from `build.sbt` we need to mark `MultiJvmPlugin` as an explicit dependency of `MultiNodePlugin`. It should have been like this anyways since `MultiNode` does use `MultiJvm`.

Reply:
The multiNode testing need be tested explicitly.
`sbt cluster/MultiJvm/test`

Reply:
> The multiNode testing need be tested explicitly. `sbt cluster/MultiJvm/test`

I don't understand the context of this comment. Are you saying that this is now a problem (i.e. the PR has changed the behaviour) or something else?

Reply:
I think you can verify it locally, the ci is not testing it.

Reply:
I will check it later.

Reply:
@He-Pin I think you have the wrong project, this is pekko-http and `cluster` is for pekko

Reply:
I also just ran the nightly tests manually and its behaving the same way as it currently does, see https://gist.github.com/mdedetrich/4e10d79f3687c49fd0886f83ec97d42a